docs/guides/commands/samples/command_handler.cs

```diff
@@ -8,36 +8,36 @@
 
 public class Program
 {
-    private CommandService commands;
-    private DiscordSocketClient client;
-    private IServiceProvider services;
+    private CommandService _commands;
+    private DiscordSocketClient _client;
+    private IServiceProvider _services;
 
     static void Main(string[] args) => new Program().Start().GetAwaiter().GetResult();
 
     public async Task Start()
     {
-        client = new DiscordSocketClient();
-        commands = new CommandService();
+        _client = new DiscordSocketClient();
+        _commands = new CommandService();
 
         string token = "bot token here";
 
-        services = new ServiceCollection()
-                .BuildServiceProvider();
+        _services = new ServiceCollection()
```

Comment:
At the very least, this could inject `_client`, and `_commands` into the container.